app/models/data_resources/open_weather_map.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class OpenWeatherMap < ApplicationRecord
+
+  store :data,
+        accessors: %i[
+          lat
+          lon
+          current
+          hourly
+          daily
+          alerts
+        ],
+        coder: JSON
+end
+
+# == Schema Information
+#
+# Table name: open_weather_maps
```

Comment:
die tabelle enthält keine verschiedenen open weather maps, sondern einzelne orte aus OpenWeaterMapdaten? versteh ich das richtig?
also wenn man einen datensatz betrahtet, hat man daten zu einer stadt? einer koordinate?
wie würdest du einen datensatz beschreiben in der tabelle?

Reply:
Jede Zeile ist ein Datenimport von OpenWeatherMap. Ein Import ist für eine Lat/lon Anfrage und liefert ein kompletten JSON Datensatz zur Wetterlage an diesem Standort. 
Ältere Imports sind dann Wetterdaten aus der Vergangenheit und werden regelmäßig gelöscht.